src/styles/theme.module.scss

```diff
@@ -0,0 +1,20 @@
+* {
+    // colors
+    --white: #ffffff;
+    --black: #000000;
+
+    --nude0: #FEF9EC;
+    --nude1: #FFEFC7;
+
+    --action2: #707365;
+
+    --ff-decor: 'Marcellus', serif;
+    --ff-default: 'Figtree', sans-serif;
```

Comment:
Where's this from? Is it what Mika provided? What does "nude" mean? "action2"? Please don't add these garbage names.

Reply:
While Mika didn't provide variables I use these temporarily. 
I'm going to change this to Mika's as soon as I get it.